Description: Add "foreign" to AC_INIT_AUTOMAKE since this doesn't
 adhere to GNU-style.
Author: Alessio Treglia <alessio@debian.org>
Forwarded: no
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
--- ois.orig/configure.ac
+++ ois/configure.ac
@@ -4,7 +4,7 @@ AC_PREREQ(2.50)
 AC_INIT( [OIS], 1.3.0 )
 
 AC_CANONICAL_TARGET
-AM_INIT_AUTOMAKE( [OIS], 1.3.0 )
+AM_INIT_AUTOMAKE([1.3.0 foreign])
 AM_CONFIG_HEADER([includes/config.h])
 
 dnl Check for programs
